motor/discovery/aws/ssm_instances.go

```diff
@@ -109,6 +109,10 @@ func (ssmi *SSMManagedInstances) getInstances(account string, ec2InstancesFilter
 				input.Filters = append(input.Filters, types.InstanceInformationStringFilter{Key: aws.String("InstanceIds"), Values: ec2InstancesFilters.InstanceIds})
 				log.Debug().Interface("instance ids", ec2InstancesFilters.InstanceIds).Msgf("filtering")
 			}
+			if ec2InstancesFilters.InstanceState == 0 {
+				input.Filters = append(input.Filters, types.InstanceInformationStringFilter{Key: aws.String("PingStatus"), Values: []string{"Online"}})
```

Comment:
i considered adding a specific filter for the ssm ping, instead of relying on the running filter.. 🤷  opinions?